_reports/RJ722.md

```diff
@@ -266,61 +267,66 @@ activity:
       - details: "Add --ignore-names flag (#139)."
 
   - 46:
+      - repo: vulture
+      - link: https://github.com/jendrikseipp/vulture/commit/3c965e2
+      - details: "Add Item.get_whitelist_string. (#144)"
```

Comment:
There are three commits in this PR, and they also need to be added to your tarball if you want to add them to the activity report.

Reply:
I've added the recent ones from Vulture. Anything else left?

Reply:
Ah I see , they are squashing into one commit.

Ok, then this checks out.  Thanks